schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+CREATE DATABASE clinicDB;
+
+/* tables from the diagrams */
+
+CREATE TABLE
+    patients(
+        id INT PRIMARY KEY,
+        name VARCHAR(250),
+        date_of_birth DATE
+    );
+
+CREATE TABLE
+    treatment(
+        id INT PRIMARY KEY,
+        name VARCHAR(250),
+    );
```

Comment:
- [ ] You did great removing the duplicate `name` field. :+1: But I believe `treatment` table has a `type` field. Please consider adding it as suggested earlier. ✍️
![image](https://github.com/Metaverse-Nabeel/HMS_DB_Design/assets/17243325/868baedd-d331-4925-a69e-fac4d98e6521)